packages/redux/src/store.ts

```diff
@@ -37,9 +37,9 @@ export interface WorkspaceSettings {
   timeZone: string;
   showTimeZone: boolean;
   showTSeparator: boolean;
-  truncateNumbersWithPound: boolean;
+  truncateNumbersWithPound?: boolean;
   disableMoveConfirmation: boolean;
-  showSystemBadge: boolean;
+  showSystemBadge?: boolean;
```

Comment:
Why are these optional now? I think they should always exist in the workspace settings

Reply:
`truncateNumbersWithPound` and `showSystemBadge` were missing from code-studio `WorkspaceSetting` under `LocalWorkspaceStorage.ts`. I ended up remove the optional from truncateNumbersWithPound and set it to false in `LocalWorkspaceStorage`, but I don't think `showSystemBadge` is used anywhere, so I made it optional

Reply:
Yes it is only used in Enterprise... so the proper way to do this would be to remove `showSystemBadge` from here, then we need to extend these types in Enterprise.